apiserver/facades/client/bundle/bundle.go

```diff
@@ -292,7 +292,7 @@ func (b *BundleAPI) fillBundleData(model description.Model) (*bundleOutput, erro
 		Machines:     make(map[string]*charm.MachineSpec),
 		Relations:    [][]string{},
 	}
-	isCaas := model.Type() == "caas"
+	isCaas := model.Type() == description.CAAS
```

Comment:
state.ModelTypeCAAS

Reply:
this model is not `state.Model`, but `description.Model`.